### DISCUSSION TITLE
Sensoren & Hardware

### DISCUSSION THREAD

**Discussion by CCernusca** [Can-Satellite-Development/Documentation-Vault#8] [Show and tell]
Hier befinden sich unsere Entschlüsse bezüglich der Hardware, genauer gesagt der Sensoren.

**Comment by CCernusca** [comment]
Unsere Hardware (**Veraltet**):

- Mikrocontroller/Minicomputer: Raspberry Pi Zero 2 (Hat hoffentlich genug Leistung & Speicher) (Bild ist Zero 1, aber 2 sieht fast gleich aus)

![20241024_163747.jpg](https://github.com/user-attachments/assets/9d4eaca4-ad66-4467-8c98-2f2cedcfc977)

- Kamera: RB-Camera-WW (https://joy-it.net/de/products/rb-camera-WW)

![20241024_163523.jpg](https://github.com/user-attachments/assets/76e5e4d6-5822-42c0-aa20-1b36a61c093c)

- Lora: Grove Lora Radio (Muss noch getestet werden)

![20241024_163556.jpg](https://github.com/user-attachments/assets/327961d7-a95b-446a-ba60-f9fa7f6f4630)

- Temperatur, Luftdruck, Luftfeuchtigkeit: BME280 (Müssen wir noch beschaffen)

- Kompasssensor: GY-273 (Müssen wir noch beschaffen)

- Akku: Realpower PB-Lipstick

![20241024_164001.jpg](https://github.com/user-attachments/assets/eeca7753-f76f-48c9-9bd7-328f3d149cc3)

![20241024_164010.jpg](https://github.com/user-attachments/assets/27113981-886a-4067-823e-3e95a7875248)

**Comment by ortanaV2** [comment]
Lipo Akku Verkäufer: https://www.otto.de/p/realpower-pb-lipstick-powerbank-2500-mah-mobiles-ladegeaet-leicht-lippenstift-design-ideal-fuer-die-handtasche-S0B1E0J6/#variationId=S0B1E0J69ENA

**Reply by ortanaV2** [reply]
Type: Li-Ion

**Reply by CCernusca** [reply]
Quelle?

**Reply by ortanaV2** [reply]
https://www.voelkner.de/products/3862217/RealPower-PB-Lipstick-Powerbank-2500-mAh-Li-Ion-Rot.html?srsltid=AfmBOop7kyTb2bWVX0YeuXCxUezMcmIe7vxa5jporwE_Wxi3VjGPFn5h

**Comment by CCernusca** [comment]
**3D-Modelle für unsere Hardware**

- [Raspberry Pi Zero 2 w](https://www.printables.com/model/988094-raspberry-pi-zero-2-w/files)
- [ESP32](https://sketchfab.com/3d-models/esp32-78c2b5a932a1463bbc6e8ada630a0545)
- [BME280](https://sketchfab.com/3d-models/bme280-from-adafruit-cansat-kit-component-d7ba810bbd724dc3957a67746bfce6fd)
- [GY-273](https://www.printables.com/model/249293-hmc5883l-i2c-compass-module/files)
- [Cam](https://www.thingiverse.com/thing:2376448/files) (Ev. Falsch)
- [Voltcraft vc 5000-2]() (Nichts gefunden)
